nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/property-verification/property-verification.component.html

```diff
@@ -0,0 +1,58 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="property-verification flex flex-col h-full gap-y-3">
+    <div class="flex justify-between items-center">
+        <div class="font-bold">Verification</div>
+        <div>
+            <button mat-icon-button color="primary" type="button" [disabled]="disabled" (click)="verifyClicked()">
+                <i class="fa fa-check"></i>
+            </button>
+        </div>
+    </div>
+    @if (disabled) {
+        <div class="verification-disabled unset">Property verification is disabled</div>
+    } @else if (isVerifying) {
+        <div class="empty">Verifying properties...</div>
+    } @else if (results.length > 0) {
+        <div class="verification-results flex flex-col gap-y-2 overflow-y-auto">
```

Comment:
IMO I think there should be a border around the verification like we do for Service references or Fetch parameters. Happy to discuss this more. Generally it looks good but my primary concern in when the results scroll like in the pic below. The border helps when the text clips.

![Screenshot 2024-05-21 at 4 14 02 PM](https://github.com/apache/nifi/assets/123395/a6bfcf0f-2c51-4c41-b224-716905a881e6)

Reply:
This makes sense. I can add a border.